packages/console/src/pages/Connectors/components/ConnectorRow/index.tsx

```diff
@@ -7,31 +7,53 @@ import Status from '@/components/Status';
 import { connectorTitlePlaceHolder } from '@/consts/connectors';
 
 import ConnectorName from '../ConnectorName';
+import * as styles from './index.module.scss';
 
 type Props = {
   type: ConnectorType;
   connector?: ConnectorDTO;
+  connectors?: ConnectorDTO[];
   onClickSetup?: () => void;
 };
 
-const ConnectorRow = ({ type, connector, onClickSetup }: Props) => {
+const ConnectorRow = ({ type, connector, connectors, onClickSetup }: Props) => {
   const { t } = useTranslation(undefined);
 
   return (
     <tr>
       <td>
-        <ConnectorName type={type} connector={connector} />
+        <ConnectorName
+          type={type}
+          connector={connector ?? connectors?.[0]}
+          isShowId={Boolean(connector)}
+        />
       </td>
       <td>{t(connectorTitlePlaceHolder[type])}</td>
       <td>
-        {type === ConnectorType.Social && (
-          <Status status={connector?.enabled ? 'enabled' : 'disabled'}>
-            {t(
-              connector?.enabled
-                ? 'admin_console.connectors.connector_status_enabled'
-                : 'admin_console.connectors.connector_status_disabled'
-            )}
-          </Status>
+        {type === ConnectorType.Social && connectors && (
+          <div className={styles.statusItems}>
+            {connectors.map(({ id, enabled, platform }) => (
+              <div key={id} className={styles.statusItem}>
+                <Status status={enabled ? 'enabled' : 'disabled'}>
+                  {t(
+                    enabled
+                      ? 'admin_console.connectors.connector_status_enabled'
+                      : 'admin_console.connectors.connector_status_disabled'
+                  )}
+                </Status>
+                <div className={styles.platform}>
+                  (
+                  {platform === ConnectorPlatform.Universal &&
+                    t('admin_console.connectors.platform.universal')}
+                  {platform === ConnectorPlatform.Web && t('admin_console.connectors.platform.web')}
+                  {platform === ConnectorPlatform.Native &&
+                    t('admin_console.connectors.platform.native')}
+                  )
```

Comment:
we could use a map to handle this.
Refer to: https://github.com/logto-io/logto/blob/f01d11344534bc82df9cfc44d2c6287c36edd0fd/packages/console/src/consts/connectors.ts

Reply:
updated